src/FormCanvas-Core/BalloonEngine.class.st

```diff
@@ -50,6 +50,13 @@ BalloonEngine class >> debug: aBoolean [
 	Debug := aBoolean
 ]
 
+{ #category : #accessing }
+BalloonEngine class >> defaultBitmapWidth [
+	"Return default bitmap width"
```

Comment:
The comment is a paraphrase og the selector. I would remove it.

Reply:
I agree with Christophe

Reply:
Ok ;-)
I will remove that.